localstack-core/localstack/services/lambda_/provider.py

```diff
@@ -1866,6 +1869,39 @@ def update_alias(
     # =======================================
     # ======= EVENT SOURCE MAPPINGS =========
     # =======================================
+    def check_service_resource_exists(self, service, resource_arn):
```

Comment:
nit: we should add type hint and a docstring (to express what exceptions it can raise) to the function
```suggestion
    def check_service_resource_exists(self, service: str, resource_arn: str) -> None:
```